docs/data/material/customization/theming/theming.md

```diff
@@ -173,6 +173,8 @@ theme = createTheme(theme, {
 
 Think of creating a theme as a two-step composition process: first, you define the basic design options; then, you'll use these design options to compose other options.
 
+**WARNING**: `theme.vars` is a private object used for CSS variables support. Please use another name for a custom object.
```

Comment:
Can you also add to https://mui.com/material-ui/customization/theming/#custom-variables before the typescript?

```md
**WARNING**: `vars` is a private object used for CSS variables support, it will throw an error if you try to:
```js
createTheme({
  vars: { ... },
})
```

```